packages/phishing-controller/src/utils.test.ts

```diff
@@ -140,3 +148,110 @@ describe('applyDiffs', () => {
     });
   });
 });
+
+describe('validateConfig', () => {
+  it('throws an error if the config is not an object', () => {
+    expect(() => validateConfig(null)).toThrow('Invalid config');
+  });
+
+  it('throws an error if the config contains a tolerance without a fuzzylist', () => {
+    expect(() => validateConfig({ tolerance: 2 })).toThrow(
+      'Fuzzylist tolerance provided without fuzzylist',
+    );
+  });
+
+  it('throws an error if the config contains an invalid name', () => {
+    expect(() => validateConfig({ name: 123 })).toThrow(
+      "Invalid config parameter: 'name'",
+    );
+  });
+
+  it('throws an error if the config contains an invalid version', () => {
+    expect(() => validateConfig({ version: { foo: 'bar' } })).toThrow(
+      "Invalid config parameter: 'version'",
+    );
+  });
```

Comment:
What do you think about adding a test for the happy path?

Reply:
Good idea! Added in [bf8fe91](https://github.com/MetaMask/core/pull/4137/commits/bf8fe913af9bd30e2b001e3d18ecd5f2a0d088f4)